<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>671</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Snap ITC</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>SiBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 219, 173);
border-color: rgb(176, 121, 70);
font: 8pt &quot;Snap ITC&quot;;</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(21, 22, 24); 
font: 8pt &quot;Sitka Heading&quot;; font-size:36pt;</string>
   </property>
   <property name="text">
    <string>Authorization</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_in_but">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_up_but">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_but">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="remember_box">
      <property name="styleSheet">
       <string notr="true">
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
      </property>
      <property name="text">
       <string>Remember me</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="show_box">
      <property name="styleSheet">
       <string notr="true">
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
      </property>
      <property name="text">
       <string>Show password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="forget_but">
      <property name="styleSheet">
       <string notr="true">font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt</string>
      </property>
      <property name="text">
       <string>Forget password?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
